invenio_jsonschemas/api.py

```diff
@@ -1,165 +1,111 @@
 # -*- coding: utf-8 -*-
```

Comment:
I think an interface on top of the Flask extension would be better than a functional interface relying on ``current_app``.

I would propose we do a proxy object ``current_schemas`` to ``current_app.extension['invenio-jsonschema']`` and add the methods in this class to that object. I.e. something a long the lines of:

```python
# ext.py

class _SchemasState(object)
    def __init__(self, app, ...)
        self.app = app

    def register_schema(self, ...):
        # ...

    def list_schemas(self):
        # ...


class InvenioJSONSchemas(object):
    # ...
    def __init__(self, app):
        if app:
            self._state = self.init_app(app)
    
    def init_app(self, app):
        _state = _SchemasState(app, schemas, ...)
        app.extensions['invenio-schemas']
        return _state

    def __getattr__(self, name):
        """Proxy to state object."""
        return getattr(self._state, name, None)


# api.py
current_schemas = LocalProxy(lambda: current_app.extensions['invenio-scheams'])
```